scripts/runfabtests.sh

```diff
@@ -355,15 +416,16 @@ function usage {
 	errcho -e " -v\tprint output of failing"
 	errcho -e " -vv\tprint output of failing/notrun"
 	errcho -e " -vvv\tprint output of failing/notrun/passing"
-	errcho -e " -t\ttest set(s): all,quick,unit,simple,standard,short (default quick)"
+	errcho -e " -t\ttest set(s): all,quick,unit,simple,standard,short,complex (default quick)"
```

Comment:
Will the 'quick' ubertest  (e.g. udp/quick.json) run as part of the quick test option?  Quick is what travis runs, correct?